main.js

```diff
@@ -0,0 +1,51 @@
+// DOM utility functions
```

Comment:
@heathervv these are some functions I thought would help for adding and remove classes. Hope this helps!